net-news/sfeed/sfeed-1.2.ebuild

```diff
@@ -12,29 +12,33 @@ SRC_URI="https://codemadness.org/releases/${PN}/${P}.tar.gz"
 LICENSE="ISC"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE="ncurses"
-
-DEPEND="ncurses? ( sys-libs/ncurses )"
-RDEPEND="${DEPEND}
-	net-misc/curl
-	sys-apps/coreutils
-	sys-libs/glibc
-	virtual/awk
-	www-client/lynx
-	x11-misc/xclip
-	x11-misc/xdg-utils"
```

Comment:
fwiw these are used by sfeed_curses/_content, lynx notably to dump html to text (although could be semi-optfeatures, not that I've tried sfeed I don't have more to say).

Reply:
same as curl, these are all optional runtime deps. I'll put them into opt-feature.